monkey/common/configuration/agent_sub_configurations.py

```diff
@@ -4,6 +4,24 @@
 
 @dataclass(frozen=True)
 class CustomPBAConfiguration:
+    """
+    Dataclass for the configuration of custom post-breach actions
+
+    Attributes:
+        linux_command (str): Command to run on Linux victim machines. If a file is uploaded,
+                             use this field to change its permissions, execute it, and/or delete
+                             it.
+                             Example: `chmod +x file.sh; ./file.sh; rm file.sh`
+        linux_filename (str): Name of the file to upload on Linux victim machines.
+                              Example: `i-am-custom-pba-file.sh`
+        windows_command (str): Command to run on Windows victim machines. If a file is uploaded,
+                               use this field to change its permissions, execute it, and/or delete
+                               it.
+                               Example: `file.bat & del file.bat`
+        windows_filename (str): Name of the file to upload on Windows victim machines.
+                                Example: `i-am-custom-pba-file.bat`
```

Comment:
This does not match the documentation format provided in our guidelines. It should be something like 
```
:param linux_command: Command to run on Linux victim machines...
```

Reply:
What guidelines?

If we accept all valid file names (per platform) then the documentation is minimal.

Reply:
Documentation format for classes. So far our guidelines explain how to document methods, but this looks like a different format. There are no examples with `Attributes:`, typehints or classes.

Reply:
I think Vakaris meant the format of the docstring. I've modified it to what we already have i.e. what we use for methods.

Reply:
It looks like dataclasses need to be handled differently. https://stackoverflow.com/questions/51125415/how-do-i-document-a-constructor-for-a-class-using-python-dataclasses

But see also https://stackoverflow.com/a/51125764/8715608